test/index.spec.ts

```diff
@@ -664,68 +663,67 @@ describe('TinyColor', () => {
   });
   it('readability', () => {
     // check return values from readability function. See isReadable above for standards tests.
-    const tinycolor = new TinyColor();
     expect(readability('#000', '#000')).toBe(1);
     expect(readability('#000', '#111')).toBe(1.1121078324840545);
     expect(readability('#000', '#fff')).toBe(21);
   });
 
   it('mostReadable', () => {
-    const tinycolor = new TinyColor();
-    expect(mostReadable('#000', ['#111', '#222']).toHexString()).toBe('#222222');
-    expect(mostReadable('#f00', ['#d00', '#0d0']).toHexString()).toBe('#00dd00');
-    expect(mostReadable('#fff', ['#fff', '#fff']).toHexString()).toBe('#ffffff');
+    expect((mostReadable('#000', ['#111', '#222']) as TinyColor).toHexString()).toBe('#222222');
```

Comment:
Why do we need to cast the return value of mostReadable as a TinyColor? I'd have thought typescript would have figured that out.

Reply:
Oh I see - it can be null. What do you think about turning off `strictNullChecks`, at least for tests? If we make a mistake and get null back when it's not expected the test will fail anyway

Reply:
@bgrins for sure, done